easybuild/easyconfigs/f/fenics-basix-py/fenics-basix-py-0.9.0-gfbf-2023b.eb

```
@@ -0,0 +1,36 @@
+# TODO: In Spack the package name is fenics-basix-py, but then the sanity checks
+# try to work with `fenics_basix_py`.
+name = 'basix'
+homepage = 'https://github.com/FEniCS/basix'
+description = "Basix is a finite element definition and tabulation runtime library"
+
+versionsuffix = ''
+
+easyblock = 'PythonPackage'
+
+# Version and source
+version = '0.9.0'
+source_urls = ['https://github.com/FEniCS/basix/archive']
+sources = ['v%(version)s.tar.gz']
+
+# Toolchain (adjust to match your target system)
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+# Build dependencies
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('scikit-build-core', '0.10.7'),
+    ('nanobind', '2.5.0'),
+    ('fenics-basix', '0.9.0')
+]
+
+# Runtime dependencies
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+]
+
+use_pip = True
+install_src = "./python"
+
```

```suggestion
modulename = "basix"

```

This option of `PythonPackage` tells EasyBuild for which name it should look in its sanity check.